examples/2d/move_sprite.rs

```diff
@@ -0,0 +1,44 @@
+use bevy::prelude::*;
+
+fn main() {
+    App::build()
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup.system())
+        .add_system(sprite_movement.system())
+        .run();
+}
+
+struct BevyLogo {
+    rising: bool,
```

Comment:
We should use an enum for this.

Reply:
I'm a bit new to rust and bevy ECS. can you explain why?

Reply:
Rust enums provide an extra level of type safety and you can also name the variants to be more descriptive. 

effectively 
```rust
enum Movement {
    Rising, 
    Falling
}
```
and 
```rust
bool
```
are equivalent. 

However the `Movement` enum is more descriptive as to what the two states actually represent.

Reply:
Especially if you call it something amusing like
`Logomotion`